.gitignore

```diff
@@ -20,7 +20,7 @@ lib/
 libreoffice/
 node_modules/
 parts/
-prototype
+/prototype
 release/
```

Comment:
This is needed otherwise git will ignore all the prototype folders (not just the one where we checkout the ploneintranet.prototype)